WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/DashboardCardsTracker.kt

```diff
@@ -0,0 +1,57 @@
+package org.wordpress.android.ui.mysite.cards.dashboard
+
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.ui.mysite.cards.dashboard.DashboardCardsTracker.DashboardCardPropertyType.POST
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.CREATE_FIRST
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.CREATE_NEXT
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.DRAFT
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType.SCHEDULED
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import javax.inject.Inject
+
+class DashboardCardsTracker @Inject constructor(
```

Comment:
Thanks for the feedback @ParaskP7 ... Sounds like a good plan. I'll drop the word. Plus I'll also create a similar tracker for items - since it won't like in the dashboard package. 👍